notebook/static/notebook/js/searchandreplace.js

```diff
@@ -166,14 +165,14 @@ define(function(require){
         .addClass("fa fa-check-square-o")
       )
       .attr('data-toggle','button')
```

Comment:
Because the button is already a toggle button, I don't think it makes sense to add a checkbox too.  I'd either need to sync the check state with the button state or keep the checkbox static.  Not to mention, on some operating systems/browsers, check boxes aren't that pretty.

The `fa-square-o` idea is easy enough though, @ellisonbg would you like that?